### PR TITLE
List: Revert styles for .dx-button .dx-icon in Generic (T1216108)

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: packages/devextreme/testing/testcafe/tests/editors/list/common.ts [hunk 1]
```diff
@@ -323,6 +324,26 @@ test('Disabled item should be focused on tab press to match accessibility criter
...
+
+  await list.focus();
+
+  await testScreenshot(t, takeScreenshot, 'The delete button should be displayed correctly after the list item focus.png');
+
+  await t
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+}).before(async () => createWidget('dxList', {
+  dataSource: {
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/testcafe/tests/editors/list/common.ts]: DataSource should be array with items. Seems, the tests testing other scenario and you add incorrect etalons.

File: packages/devextreme/testing/testcafe/tests/editors/list/common.ts [hunk 2]
```diff
@@ -323,6 +324,26 @@ test('Disabled item should be focused on tab press to match accessibility criter
   searchEnabled: true,
 }));
 
+test('The delete button should be displayed correctly after the list item focus (T1216108)', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const list = new List('#container');
+
+  await list.focus();
+
+  await testScreenshot(t, takeScreenshot, 'The delete button should be displayed correctly after the list item focus.png');
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/testcafe/tests/editors/list/common.ts]:
```suggestion
  await testScreenshot(t, takeScreenshot, 'The delete button should be displayed correctly after the list item is focused.png');
```

File: packages/devextreme/scss/widgets/generic/list/_index.scss [hunk 3]
```diff
@@ -118,6 +118,17 @@ $generic-list-search-editor-height: round($generic-base-line-height * $generic-b
...
     @include toggle-delete-icon($list-item-active-color);
   }
+
+  .dx-list-static-delete-button .dx-icon {
+    color: $list-icon-color;
+  }
+
+  &.dx-state-active,
+  &.dx-state-focused {
+    .dx-list-static-delete-button .dx-icon {
```
Comment (EugeniyKiyashko) [inline packages/devextreme/scss/widgets/generic/list/_index.scss:128]: The problem can also arise not only for the delete button but also for any other buttons that may be added by the user using a template.

File: packages/devextreme/testing/testcafe/tests/editors/list/common.ts (hunk 1 above)
Comment (marker-dao) [reply packages/devextreme/testing/testcafe/tests/editors/list/common.ts]: [dc8336e2178a7eee3781518b0c9a0ee2bf691497]

File: packages/devextreme/testing/testcafe/tests/editors/list/common.ts (hunk 2 above)
Comment (marker-dao) [reply packages/devextreme/testing/testcafe/tests/editors/list/common.ts]: [a57527cfd2bbfb22c9e7e1148e61b6811d1283e4]

**Review by EugeniyKiyashko — DISMISSED** [review]

**Review by Zedwag — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**